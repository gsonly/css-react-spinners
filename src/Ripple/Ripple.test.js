import React from 'react'
import renderer from 'react-test-renderer'
import { Ripple } from '..'

describe('Ripple', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Ripple />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
