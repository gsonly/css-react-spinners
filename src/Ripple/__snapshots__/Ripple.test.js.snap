// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ripple renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="lds-ripple"
  >
    <div />
    <div />
  </div>,
]
`;
